Makefile

```
@@ -233,25 +213,21 @@ golangci-lint: $(EMBEDDED_YAMLS)
 unit: $(EMBEDDED_YAMLS)
 
 # Operator SDK
-# On version bumps, the checksum will need to be updated manually.
-# If necessary, the verification *keys* can be updated as follows:
-# * update scripts/operator-sdk-signing-key.asc, import the relevant key,
-#   and export it with
-#     gpg --armor --export-options export-minimal --export \
-#     ${fingerprint} >> scripts/operator-sdk-signing-key.asc
-#   (replacing ${fingerprint} with the full fingerprint);
-# * to update scripts/operator-sdk-signing-keyring.gpg, run
-#     gpg --no-options -q --batch --no-default-keyring \
-#     --output scripts/operator-sdk-signing-keyring.gpg \
-#     --dearmor scripts/operator-sdk-signing-key.asc
 $(OPERATOR_SDK):
-	curl -Lo $@ "https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk-v${OPERATOR_SDK_VERSION}-x86_64-linux-gnu"
-	curl -Lo $@.asc "https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk-v${OPERATOR_SDK_VERSION}-x86_64-linux-gnu.asc"
-	gpgv --keyring scripts/operator-sdk-signing-keyring.gpg $@.asc $@
-	sha256sum -c scripts/operator-sdk.sha256
+	mkdir -p bin && \
+	cd bin && \
```

Don’t hard-code the directory:

```suggestion
	mkdir -p $(@D) && \
	cd $(@D) && \
```